### ISSUE TITLE
retrain with InceptionNet V3 but not converge with low learning rate

### ISSUE THREAD

**Issue by lunasdejavu** [tensorflow/models#1349]
My enviroment is ubuntu 16.04 x64 tensorflow 1.0.0 by pip
with  python2.7
only one GPU 1070 RAM 8G CPU.
Cuda compilation tools, release 8.0, V8.0.61
bazelBuild label: 0.4.5
cuDNN v5.1 
i7 RAM8G
the top folder is /home/lunasdejavu/Downloads/Inception/
I read the tutorial already, then I tried to train a dataset which is consist of 10382 images for training , 1154 images for validation and only 2 class.
I am very new to python so after I turned the dataset to 1 shards `TFRecords` because the tutorial said 1024 for 1 shard.
the I changed  `enum_classes()` and `num_examples_per_epoch()` in `flowers_data.py`
,`FLOWERS_DATA_DIR=/tmp/my-custom-data/`
the question is that the loss  never converges even after 100000
i tried 0.1 0.05 would be  too big that the loss could be bigger than the  loss of the first step
then after that 0.01 0.001 is always look like this
[http://imgur.com/a/kD4Fi](url)
the learnig rate seemed to drop too fast ,and the loss looked the same after many steps
and I read  a few papers said that the change of the loss between N step and N+1 step should be less than 0.01. 
the learning rate is 0.0005 now
so should I make it lower like 0.0001 now?

------------------------

### System information
- **What is the top-level directory of the model you are using**:
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script: https://github.com/tensorflow/tensorflow/tree/master/tools

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem

Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by jart** [comment]
This might be a better question for [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow). Please see our issue tracker [policy](https://github.com/tensorflow/tensorflow/issues/new).
